docs/versions/latest/Release.md

```diff
@@ -190,6 +190,17 @@ The `enableConfigManager`, `maximumHardwareVersionKey` and `desiredSoftwareSpec`
 
 <https://github.com/vmware/build-tools-for-vmware-aria/issues/297>
 
+### Cloud Template folder name and details.json->name mismatch issue fixed
+
+#### Previous Behavior
+
+- When Cloud-Template folder name and details.json->name mismatch,  folder name is used for validating if the record exists on the server and details.json->name is used for creating/updating the cloud template on server.
```

Comment:
Use of '->' is confusing, please reword - e.g. 'When the Cloud-Template folder name does not match the name from details.json, ...'. Same for current behaviour and the comment in VraNgBlueprintStore.java (l.245)